scripts/ci/Dockerfile

```diff
@@ -148,19 +151,31 @@ COPY --chown=pwuser:pwuser ${APP_HOME}/entrypoint.sh .
 ENTRYPOINT ["./entrypoint.sh"]
 
 
-FROM playwright-base as output-local
-ARG APP_HOME
 
 
-# node user exists in the base image
-RUN mkdir -p /out \
-  && chown node:node /out
+FROM base-image as localrun
+WORKDIR /data
 
-COPY --chown=pwuser:pwuser ${APP_HOME}/entrypoint.sh .
+RUN wget -O /tmp/jq-linux64 https://github.com/stedolan/jq/releases/download/jq-1.6/jq-linux64 && \
+  chmod a+x /tmp/jq-linux64 && \
+  mv /tmp/jq-linux64 /usr/bin/jq
+
+RUN apk add --update --no-cache git bash nodejs python3 build-base gcc openjdk11 && ln -sf /usr/bin/python3 /usr/bin/python
+RUN ["yarn", "set", "version", "berry"]
+# ENV YARN_CACHE_FOLDER=/cache/yarn
+# RUN mkdir -p $YARN_CACHE_FOLDER
```

Comment:
Unnecessary comments or something you want to implement later?

Reply:
I want to implement caching in the container later. Preferably even somehow overlay outputs so we never touch the local directory.